Library/Formula/repo.rb

```diff
@@ -3,9 +3,9 @@
 class Repo < Formula
   desc "Repository tool for Android development"
   homepage 'http://source.android.com/source/git-repo.html'
-  url 'https://raw.githubusercontent.com/android/tools_repo/v1.12.13/repo'
+  url 'https://raw.githubusercontent.com/android/tools_repo/v1.12.21/repo'
   version '1.21'
```

Comment:
The stable version currently is 1.21 but the url was pointing to incorrect
version of 1.13

On Sunday, June 14, 2015, Dominyk Tiller <notifications@github.com> wrote:

> In Library/Formula/repo.rb
> <https://github.com/Homebrew/homebrew/pull/40705#discussion_r32382597>:
>
> > @@ -3,9 +3,9 @@
> >  class Repo < Formula
> >    desc "Repository tool for Android development"
> >    homepage 'http://source.android.com/source/git-repo.html'
> > -  url 'https://raw.githubusercontent.com/android/tools_repo/v1.12.13/repo'
> > +  url 'https://raw.githubusercontent.com/android/tools_repo/v1.12.21/repo'
> >    version '1.21'
>
> Presumably this needs bumping as well?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Homebrew/homebrew/pull/40705/files#r32382597>.
>